plugins/gatsby-source-jenkinsplugins/utils.js

```diff
@@ -115,6 +115,20 @@ const fetchPluginData = async ({createNode, reporter}) => {
                         contentDigest: crypto.createHash('md5').update(`plugin${pluginData.name.trim()}`).digest('hex')
                     }
                 });
+                pluginData.dependencies.forEach(dependency => {
+                    const mergedId = `${pluginData.name.trim()}:${dependency.name.trim()}`;
+                    createNode({
```

Comment:
so createNode returns a promise (https://www.gatsbyjs.com/docs/actions/#createNode) that means all these dependency nodes won't be waiting until completion.


```suggestions
                createNode().then(pluginNode => Promise.all(pluginData.dependencies.map(dependency => {
                    const mergedId = `${pluginData.name.trim()}:${dependency.name.trim()}`;
                    return createNode({});
                  });
```